app/public/js/profile_save_prompt.js

```diff
@@ -1,12 +1,9 @@
 $(window).on('beforeunload', function(e) {
```

Comment:
Not related to this file but get latest.  

See "Say your fork's master branch is behind the main repository's master" on:
https://github.com/hhaccessibility/hhaccessibility.github.io/wiki/git-commands